src/inputs/input-with-icon.scss

```diff
@@ -0,0 +1,35 @@
+// Copyright (c) Bentley Systems, Incorporated. All rights reserved.
+// See LICENSE.md in the project root for license terms and full copyright notice.
+@import '../style/index';
+@import './input';
+
+// Mixin to have inputs combined with icon (eg. select)
+@mixin iui-input-with-icon {
+  display: grid;
+  align-items: center;
+  cursor: pointer;
+
+  > :first-child {
+    grid-area: 1 / -1;
+    padding-right: $iui-icons-default + $iui-l;
+  }
+
+  &:focus-within {
+    > :first-child {
+      outline: 2px solid var(--_focus-color, var(--iui-color-foreground-primary));
+      outline-offset: -2px;
+    }
+  }
+
+  &:hover {
+    > :first-child {
+      border-color: var(--_hover-color, rgba(t(iui-color-foreground-body-rgb), t(iui-opacity-2)));
```

Comment:
Interesting, I've never seen a CSS variable like this before.  Can you explain what this is doing?

Reply:
Can you check if it even works? we should not use `t()` without `@include themed` ...?

Reply:
It's just using the CSS variable `--_hover-color`. If that variable is not found, then it's using the default value.

We set that variable in select.scss

Reply:
It does work, I think `t()` just converts to `var()` if not inside themed mixin. But you're right, we should change it to `var()`